src/Tabs/Tabs.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+export const Tabs = ({ tabs, selected, onChange }) => {
+    const chosenTab = tabs.find(tab => tab.id === selected);
+
+    return (
+      <div>
+        <ul className="nav nav-tabs">
+          {
+            tabs.map( tab=> (
+            <li
+              className={`nav-item nav-link ${tab.id === selected && 'active font-weight-bold'}`}
```

Comment:
use classnames library
https://www.npmjs.com/package/classnames

Reply:
thanks, somehow I've forgotten about it.